### PR TITLE
Added docs for Page and PageContent

### PR DESCRIPTION
**Pull request by jcfilben**
related to https://github.com/grommet/hpe-design-system/issues/2334

### CODE REVIEW

File: src/screens/Page.js [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+          <PropertyValue type="object">
+            <Example defaultValue>{`{ pad: { horizontal: 'large' }}`}</Example>
+          </PropertyValue>
+        </Property>
+      </ThemeDoc>
+    </ComponentDoc>
+
+    <ComponentDoc
+      name="PageContent"
+      description="An container for content within a Page"
```
Comment (ericsoderberghp) [inline src/screens/Page.js:191]:
```suggestion
      description="A container for content within a Page"
```

File: src/screens/Page.js [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+        path: '/box',
+        defaultProps: {
+          fill: 'horizontal',
+        },
+      }}
+    >
+      <Properties>
+        <Property name="background">
+          <Description>
+            The background of the Page. Use `full: "horizontal"` to specify that
```
Comment (ericsoderberghp) [inline src/screens/Page.js:204]:
```suggestion
            The background of the Page. Use `fill: "horizontal"` to specify that
```

File: src/screens/Page.js [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+              {`
+{
+  color: "neutral-1",
+  dark: true,
+  opacity: true,
+  position: "bottom",
+  repeat: "no-repeat",
+  size: "cover",
+  image: "url(//my.com/assets/img.png)",
+  full: "horizontal
```
Comment (ericsoderberghp) [inline src/screens/Page.js:224]:
```suggestion
  fill: "horizontal
```

Comment (jcfilben) [reply src/screens/Page.js:224]: Thanks for catching these! Going back and fixing...

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**